third_party/amd/lib/TritonAMDGPUTransforms/CanonicalizePointers.cpp

```diff
@@ -0,0 +1,432 @@
+/*
+ * Copyright (c) 2023 NVIDIA Corporation & Affiliates. All rights reserved.
```

Comment:
For AMD specific pass, we don't have this copyright thing

Reply:
Sure, I removed it